extensions/git/src/repository.ts

```diff
@@ -170,6 +170,7 @@ export class Resource implements SourceControlResourceState {
 	get letter(): string {
 		switch (this.type) {
 			case Status.INDEX_MODIFIED:
+				return 'S';
```

Comment:
I don't think this is a good idea, since we've been using `M` for this for a long long time.